message_definitions/v1.0/common.xml

```diff
@@ -3495,7 +3495,7 @@
       <field type="int32_t" name="baseline_a_mm" units="mm">Current baseline in ECEF x or NED north component in mm.</field>
       <field type="int32_t" name="baseline_b_mm" units="mm">Current baseline in ECEF y or NED east component in mm.</field>
       <field type="int32_t" name="baseline_c_mm" units="mm">Current baseline in ECEF z or NED down component in mm.</field>
-      <field type="uint32_t" name="accuracy">Current estimate of baseline accuracy.</field>
+      <field type="uint32_t" name="accuracy" units="c%">Current estimate of baseline accuracy</field>
```

Comment:
Again percentage for an accuracy number on a GPS is illogical. We're talking about a device that measures our real world position, what is 100% accuracy, and what's 1% accuracy mean?